app/models/guide.rb

```diff
@@ -50,6 +50,15 @@ def has_published_edition?
     editions.where(state: "published").any?
   end
 
+  def has_unpublished_edition?
+    editions.where(state: "unpublished").any?
+  end
```

Comment:
The other method is used outside the class, which is why I kept them both
right next to each other, more consistent.

On Tuesday, 26 April 2016, Henry <notifications@github.com> wrote:

> In app/models/guide.rb
> <https://github.com/alphagov/service-manual-publisher/pull/239#discussion_r61126372>
> :
>
> > @@ -50,6 +50,15 @@ def has_published_edition?
> >      editions.where(state: "published").any?
> >    end
> >
> > +  def has_unpublished_edition?
> > +    editions.where(state: "unpublished").any?
> > +  end
>
> Oh.. could they both be moved to private then? If we're using the methods
> anywhere outside the class they should be private. Right?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/alphagov/service-manual-publisher/pull/239/files/81509c3e39e86e56ed610324cb908fa8640eabfd#r61126372>
>